map.go

```diff
@@ -61,6 +61,15 @@ func NewMapFromKeys(keys Set, f func(key interface{}) interface{}) Map {
 	return b.Finish()
 }
 
+// ToFrozenMap takes a map[interface{}]interface{} and returns a frozen Map from it.
+func ToFrozenMap(m map[interface{}]interface{}) Map {
+	keys := NewSetBuilder(len(m))
```

Comment:
Why not build the map directly via `NewMapBuilder`?

Reply:
Thought it was nicer to just use `NewMapFromKeys`, just realized it's doing unnecessary work. The change is added.